draft-ietf-dnsop-dns-catalog-zones.md

```diff
@@ -498,9 +498,38 @@ before accepting a member zone).
 With migration of member zones between catalogs using the `coo` property, it is possible for the owner of the target catalog (i.e., `$NEWCATZ`) to take over all its associated state with the zone from the original owner (i.e., `$OLDCATZ`) by maintaining the same member node label (i.e., `<unique-N>`).
 To prevent the takeover of the zone associated state, the original owner has to enforce a zone state reset by changing the member node label (see (#zonereset)) before or simultaneously with adding the `coo` property.
 
-# IANA Considerations
+# IANA Considerations {#iana}
 
-This document has no IANA actions.
+IANA is requested to create a registry on the "Domain Name System (DNS) Parameters" IANA web page as follows:
+
+Registry Name:
+: DNS Catalog Zones Properties
+
+Assignment Policy:
+: Expert Review
+
+Reference:
+: [this document]
+
+Property prefix | Description         | Status          |Reference
+----------------|---------------------|-----------------|----------------
+version         | Schema version      | Standards Track | [this document]
+coo             | Change of Ownership | Standards Track | [this document]
+group           | Group               | Standards Track | [this document]
```

Comment:
```suggestion
group           | Group               | Standards Track | [this document]
ext             | Custom properties   | Standards Track | [this document]
```

Reply:
I think ext shoud be *.ext

Reply:
> I think ext shoud be *.ext

I think I agree. I've added a suggestion for that, and another one stating that property prefixes ending in the label "ext" are for Private Use.

@mind04 @Habbie @peterthomassen WDYT?

Reply:
sounds good!

Reply:
Okay, I committed the suggestions because I want to add the Note too. @mind04 and @Habbie can still review and comment on the PR if they disagree or have a better suggestion.